cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -422,6 +446,19 @@ def deserialize(self, proto: v2.program_pb2.Program) -> cirq.Circuit:
                     deserialized_constants.append(circuit.freeze())
                 elif which_const == 'qubit':
                     deserialized_constants.append(v2.qubit_from_proto_id(constant.qubit.id))
+                elif which_const == 'operation_value':
+                    deserialized_constants.append(
+                        self._deserialize_gate_op(
+                            constant.operation_value,
+                            arg_function_language=arg_func_language,
+                            constants=proto.constants,
+                            deserialized_constants=deserialized_constants,
+                        )
+                    )
+                elif which_const == 'moment_value':
```

Comment:
WDYT of adding a catch-all `else` to handle cases where a new value was added but this block wasn't updated? It looks like we would silently ignore those cases today.